src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -210,8 +210,10 @@ public function get_view_html() {
 			return $pre_html;
 		}
 
+		$is_single_event = 'single-event' === $view_slug || ( $query->is_singular && $query->tribe_is_event_query );
```

Comment:
When we get here, `$view_slug` has defaulted to "month" for some reason yet `$query->is_singular `& `$query->tribe_is_event_query` are both true.